### PR TITLE
Fix pinning

### PR DESCRIPTION
**Pull request by MasterWayZ**
The one changed to master does not have any of its own commits.

### CODE REVIEW

File: modules/roles_profiles/manifests/profiles/gecko_3_t_osx_1014_generic_worker.pp [hunk 1]
```diff
@@ -21,9 +21,9 @@
                 telegraf_user       => lookup('telegraf.user'),
                 telegraf_password   => lookup('telegraf.password'),
                 puppet_env          => 'dev',
-                puppet_repo         => 'https://github.com/davehouse/ronin_puppet.git',
-                puppet_branch       => '1561956_generic-worker_15-recover',
-                puppet_notify_email => 'dhouse@mozilla.com',
+                puppet_repo         => 'https://github.com/mozilla-platform-ops/ronin_puppet.git',
+                puppet_branch       => 'master',
```
Comment (markcor) [inline modules/roles_profiles/manifests/profiles/gecko_3_t_osx_1014_generic_worker.pp:25]: With the builder branch from Dave's repo, there was 100+ commits difference. It maybe worth checking before pinning to master on this repo.

File: modules/roles_profiles/manifests/profiles/gecko_1_b_osx_1015_generic_worker.pp [hunk 2]
```diff
@@ -21,9 +21,9 @@
                 telegraf_user       => lookup('telegraf.user'),
                 telegraf_password   => lookup('telegraf.password'),
                 puppet_env          => 'dev',
-                puppet_repo         => 'https://github.com/davehouse/ronin_puppet.git',
+                puppet_repo         => 'https://github.com/mozilla-platform-ops/ronin_puppet.git',
                 puppet_branch       => 'bug1665379_mac-builders',
-                puppet_notify_email => 'dhouse@mozilla.com',
+                puppet_notify_email => 'relops@mozilla.com',
```
Comment (markcor) [inline modules/roles_profiles/manifests/profiles/gecko_1_b_osx_1015_generic_worker.pp:26]: If there is significant volume, let's move this to a different google group.

**Review by markcor — CHANGES_REQUESTED** [review]

File: modules/roles_profiles/manifests/profiles/gecko_3_t_osx_1014_generic_worker.pp (hunk 1 above)
Comment (MasterWayZ) [reply modules/roles_profiles/manifests/profiles/gecko_3_t_osx_1014_generic_worker.pp:25]: If you look at https://github.com/mozilla-platform-ops/ronin_puppet/tree/1561956_generic-worker_15-recover it just says `588 commits behind master` with nothing ahead.

File: modules/roles_profiles/manifests/profiles/gecko_1_b_osx_1015_generic_worker.pp (hunk 2 above)
Comment (MasterWayZ) [reply modules/roles_profiles/manifests/profiles/gecko_1_b_osx_1015_generic_worker.pp:26]: I don't know honestly, I'll make a group.

File: modules/roles_profiles/manifests/profiles/gecko_3_t_osx_1014_generic_worker.pp (hunk 1 above)
Comment (markcor) [reply modules/roles_profiles/manifests/profiles/gecko_3_t_osx_1014_generic_worker.pp:25]: Can we test the a Mac tester against the master branch before merging this? That is such a large difference there is maybe something that breaks.

Comment (MasterWayZ) [reply modules/roles_profiles/manifests/profiles/gecko_3_t_osx_1014_generic_worker.pp:25]: True. We can do that or not move to master and leave it as-is.

Comment (MasterWayZ) [reply modules/roles_profiles/manifests/profiles/gecko_3_t_osx_1014_generic_worker.pp:25]: Let's leave it as-is and not make our lives extra hard.

**Review by markcor — APPROVED** [review]

**Pull request merged**